content/en/blog/_posts/2023-08-04-kubernetes-1.28-blog.md

```diff
@@ -0,0 +1,270 @@
+
+## Changes to supported skew between control plane and node versions
+    
+This enables testing and expanding the supported skew between core node and control plane components by one version from n-2 to n-3, so that node components (kubelet and kube-proxy) for the oldest supported minor version work with control plane components (kube-apiserver, kube-scheduler, kube-controller-manager, cloud-controller-manager) for the newest supported minor version.
+    
+The Kubernetes yearly support period already makes annual upgrades possible. Users can upgrade to the latest patch versions to pick up security fixes and do 3 sequential minor version upgrades once a year to "catch up" to the latest supported minor version.
+
+However, since the tested/supported skew between nodes and control planes is currently limited to 2 versions, a 3-version upgrade would have to update nodes twice to stay within the supported skew.
+
+## Beta support for enabling swap space on Linux
+    
+This adds swap support to nodes in a controlled, predictable manner so that Kubernetes users can perform testing and provide data to continue building cluster capabilities on top of swap.
+
+There are two distinct types of users for swap, who may overlap:
+
+- Node administrators, who may want swap available for node-level performance tuning and stability/reducing noisy neighbor issues.
+    
+- Application developers, who have written applications that would benefit from using swap memory. There are hence a number of possible ways that one could envision swap use on a node.
+    
+## Mixed version proxy (alpha) {#mixed-version-proxy}
+
+When a cluster has multiple API servers at mixed versions (such as during an upgrade/downgrade or when runtime-config changes and a rollout happens), not every apiserver can serve every resource at every version.
+
+For Kubernetes v1.28, you can enable the _mixed version proxy_ within the API server's aggregation layer.
+The mixed version proxy finds requests that the local API server doesn't recognize but another API server
+inside the control plan is able to support. Having found a suitable peer, the aggregation layer proxies
+the request to a compatible API server; this is transparent from the client's perspective.
+    
+When an upgrade or downgrade is performed on a cluster, for some period of time the API servers
+within the control plane may be at differing versions; when that happens, different subsets of the
+API servers are able to serve different sets of built-in resources (different groups, versions, and resources
+are all possible). This new alpha mechanism lets you hide that skew from clients.
+
+## CRD validation ratcheting
+    
+This Allows CRs to fail validation if the patch did not alter any of the invalid fields.  The ability to shift left validation logic from controllers to the front end is a long-term goal for improving the useability of the Kubernetes project.
```

Comment:
This text needs work, I think.

- “custom resources” not “CRs”
- it's not that Kubernetes allows CRs to fail validation; it's that changes to CRs can now pass admission-time checks, even when there are (certain kinds of) validation failures.

Reply:
I updated the CR's to custom resources. Do you think it is better if we just take this whole section out of the blog?